forms/src/main/java/swim/basic/BasicPlane.java

```diff
@@ -0,0 +1,26 @@
+package swim.basic;
+
+import swim.actor.ActorSpace;
+import swim.api.SwimRoute;
+import swim.api.agent.AgentRoute;
+import swim.api.plane.AbstractPlane;
+import swim.api.space.Space;
+import swim.kernel.Kernel;
+import swim.server.ServerLoader;
+import swim.structure.Value;
+
+public class BasicPlane extends AbstractPlane {
+
+  @SwimRoute("/unit/:id")
+  AgentRoute<UnitAgent> unitAgentType;
+
+  public static void main(String[] args) {
```

Comment:
I like @ajay-gov 's suggestion about first exercising `Form`s without the Swim stack. Will comment on #7 about how to best approach this.